terraform/terraform.tf

```diff
@@ -39,4 +39,5 @@ module paas {
   log_url                     = var.paas_log_url
   docker_credentials          = local.docker_credentials
   app_secrets_variable        = local.app_secrets
+  app_config_variable         = local.app_config
```

Comment:
you could `merge` the two and pass as app_env_variables to the paas module

Reply:
the merge is done in the module itself...